ci/docker/linux_opensuse_tumbleweed

```diff
@@ -0,0 +1,19 @@
+FROM opensuse/tumbleweed
+RUN \
+  zypper -n update && \
+  zypper -n install \
+  shadow util-linux util-linux-systemd systemd-devel \
+  git make gcc gcc-c++ perl-IPC-Run \
+  ccache clang meson ninja bison flex diff gettext-runtime \
+  llvm-devel icu libicu-devel liblz4-devel libopenssl-devel \
+  readline-devel pam-devel tcl-devel python3-devel \
+  libxml2-devel libxslt-devel libldapcpp-devel libuuid-devel \
+  krb5-devel krb5-client krb5-server krb5-plugin-kdb-ldap \
+  openldap2 openldap2-client
+# Fixing libperl.so not found error
+RUN \
+  find /usr/lib/ -name libperl.so -exec ln -sf '{}' /usr/lib64/libperl.so ';'
+# Fixing ICU errors caused by locale being set to 'POSIX' 
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
```

Comment:
Hm, odd that that's required. Is it even for the autoconf build?

The different ENV lines should probably be combined so that they result in one docker layer instead of three...

Reply:
No, autoconf was working correctly but meson was failing with
`ERROR:  collations with different collate and ctype values are not supported by ICU
CONTEXT:  SQL statement "CREATE COLLATION test1 (provider = icu, lc_collate = 'C', lc_ctype = 'C.UTF-8');"
PL/pgSQL function inline_code_block line 3 at EXECUTE` error while running `pg_regress/collate.linux.utf8` test.